mautic/mautic
# [UI] Remove all background color classes

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🟢
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR removes all background classes to make sure everything will inherit the `<body>` standard, increasing the compatibility with the dark theme, while simplifying our way of building the UI. 

It means we can improve consistency by defining properties according to the elements instead of using separate classes. It also makes easier to maintain the project, since new CSS features can handle different scenarios automatically. More focus on features, less focus on building the interface.

Tested all main pages, they seem fine.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open every page and check if everything is displaying fine
